cleanlab/datalab/issue_finder.py

```diff
@@ -182,12 +182,29 @@ def find_issues(
                 print(f"Error in {issue_manager.issue_name}: {e}")
                 failed_managers.append(issue_manager)
 
+        # Run imagelab
+        try:
+            if self.verbosity:
+                print(
+                    f'Finding {", ".join(issue_types_copy["image_issue_types"].keys())} images ...'
+                )
+            self.imagelab.find_issues(issue_types=issue_types_copy["image_issue_types"])
+            data_issues.collect_statistics_from_issue_manager(self.imagelab)
+            data_issues._collect_results_from_imagelab(self.imagelab)
+        except Exception as e:
+            print(f"Error in checking for image issues: {e}")
+            failed_managers.append(self.imagelab)
+
```

Comment:
This try-block is nearly identical to the one in the for-loop above, except:

- `issue_manager` becomes `self.imagelab`.

If we make adapter classes to these methods (in `DataIssues`). Then we can just make an adpter class for IssueFinder as well, that only needs to define `find_issues`.